tests/services/test_demo_cli.py

```diff
@@ -53,7 +53,7 @@ def _assert_single_hit(response, expected_record):
     _assert_fields(root_fields, search_hit, expected_record)
 
     access_fields = [
-        "metadata", "files", "owned_by", "access_right", "embargo_date"
+        "record", "files", "owned_by", "embargo", "grants"
```

Comment:
Thanks! Although the test is being skipped (I don't know why that was done), we should unskip it eventually (don't worry about it for now.